### DISCUSSION TITLE
alert support for offshore marine zones?

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#298] [Q&A]
Are there plans to include the offshore marine zones under /alerts within api.weather.gov?

E.g., currently there are hurricane warnings for offshore Gulf of Mexico zones (e.g. GMZ013, GMZ015) but the alerts are not returned via queries like https://api.weather.gov/alerts/active/zone/GMZ015 or https://api.weather.gov/alerts/active/region/GM

Thank you!

**Comment by formula4** [comment]
I would like to see this as well.

**Comment by scadergit** [comment]
Yes, that is on the roadmap. Don't have a date just yet, sorry.
